community.egi.eu.yml

```diff
@@ -3,6 +3,15 @@
 - hosts: forum
   name: harden ssh
   become: true
+  tasks:
+    - name: Configure remote access
+      authorized_key:
+        user: root
+        exclusive: true
+      with_file:
```

Comment:
this should probably be done using a loop on item using the url lookup plugin : https://docs.ansible.com/ansible/latest/plugins/lookup/url.html
Because those are not files :-/

Reply:
Sorry I mixed the `key` and `file` options.
Should be better now.

Reply:
I stand to be corrected, but that will probably fail too, because you're passing a loop with three strings, instead of the _contents_ of urls (which is what [the url lookup](https://docs.ansible.com/ansible/latest/plugins/lookup/url.html) should do.

Reply:
Yes, it's one purpose: https://docs.ansible.com/ansible/latest/modules/authorized_key_module.html#parameters

Reply:
Ah, I do apologise! you are indeed correct.